README.adoc

```diff
@@ -0,0 +1,24 @@
+= Schulich Delta Client Telemetry
+
+This repository is owned by the University of Calgary Solar Car team. 
+This program receives data through a serial port from the Schulich Delta. 
+This program was created using the Qt framework in conjunction with QwtPlot. 
+The master branch is compatible with Qt 5.5.1 and QwtPlot 6.1.2.
+
+== Installing Qt for Linux 
+
+Install the source at http://www.qt.io/download-open-source/ and follow the steps in the installer.
+
+== Installing QwtPlot for Linux
+
+.	Download the source at http://sourceforge.net/projects/qwt/
+.	Extract using `tar -xvjf qwt-6.1.2` 
+.	Copy the folder using `sudo cp -r qwt-6.1.2 /usr/local/qwt-6.1.2`
```

Comment:
You don't need to copy the folder there, also you shouldn't. That will just put unneeded stuff within your usr/local directory. Copy the folder to where ever and just use sudo make install. That will copy the required stuff over to that location.